lib/Onnxifi/GlowOnnxifiManager.h

```diff
@@ -116,8 +117,11 @@ class GlowOnnxifiManager final {
   /// The set of all valid glow Graphs.
   std::unordered_set<GraphPtr> graphs_;
 
-  /// Map from BackendKind to HostManager managing devices of that kind.
-  std::map<BackendKind, std::unique_ptr<runtime::HostManager>> hostManagers_;
+  /// Map from BackendKind to HostManager managing devices of that kind that is
+  /// shared by all BackendIds using that HostManager. HostManager is stored as
+  /// weak_ptr here so that it will be destructed when the last BackendId using
+  /// it is destroyed not when this singleton is destroyed.
```

Comment:
This only comes into effect if the GlowOnnxifiManager outlives all Backends, is that what we expect?

Reply:
It always will, it's a static singleton